docs/blueprints/EP035.rst

```diff
@@ -0,0 +1,140 @@
+:EP: 35
+:Title: Kytos stats with OF table stats
+:Authors:
+    - Gretel Liz De la Peña <gdelapea AT fiu DOT edu>
+    - Italo Valcy <idasilva AT fiu DOT edu>
+    - Vinicius Arcanjo <vindasil AT fiu DOT edu>
+:Created: 2023-03-13
+:Kytos-Version: 2023.1
+:Status: Draft
+
+***************************************
+EP035 - Kytos stats with OF table stats
+***************************************
+
+Abstract
+========
+
+This blueprint is to support table statistics and concentrate the statistics into a single NApp.
+
+Actions will involve ``flow_stats``, ``of_core`` and Zabbix Wrapper.
+
+Motivation
+==========
+
+Currently ``flow_stats`` is responsible to deal with flow statistics, which are exposed in endpoints:
+
+- ``GET v1/flow/stats`` to get the flows stats by dpid.
+
+- ``GET v1/packet_count/<flow_id>`` for packet count of a specific flow.
+
+- ``GET v1/bytes_count/<flow_id>`` for bytes count of a specific flow.
+
+- ``GET v1/packet_count/per_flow/<dpid>`` for packet count per flow.
+
+- ``GET v1/bytes_count/per_flow/<dpid>`` for bytes count per flow.
+
+However, OF table stats are not supported. 
+
+The goal of this blueprint is to evolve ``flow_stats`` into ``kytos_stats`` for kytos statistics, including OF table stats support.
+
+Specification
+=============
+
+Information about tables is requested with the ``OFPMP_TABLE`` type of the ``OFPT_MULTIPART_REQUEST`` message. 
+The request does not contain any data in the body and the body of the reply consists of an array of struct ``ofp_table_stats``, which contains:
+
+- `table_id`: Identifier of table.
+
+- `active_count`: Number of active entries.
+
+- `lookup_count`: Number of packets looked up in table.
+
+- `matched_count`: Number of packets that hit table. 
+
+Proposed Changes
+----------------
+
+- Create the new napp ``kytos_stats`` from ``flow_stats``. 
+- New event from ``of_core``.
+- Update ``kytos_stats`` to suport OF table stats.
+- Update Zabbix Wrapper.
+- Update the routes in the e2e test suite for `flow_stats` to `kytos_stats`. 
```

Comment:
Let's try to also plan to include at least one e2e for the new endpoint `v1/table/stats` using OvS too. Just so we can have regression tests with mininet too.

Reply:
Cool. Let's close this thread here then. Thanks.

Reply:
Thank You.